### ISSUE TITLE
Highlight binary files appropriately

### ISSUE THREAD

**Issue by alex** [so-fancy/diff-so-fancy#155]
Normal diff sections are nicely formatted like:

```
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
modified: administration/project_summary.txt
──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
```

with bright white text. However binary files have a far more plain, and easily missed:

```
Binary files a/administration/priorities.pdf and b/administration/priorities.pdf differ
```

in boring gray.

It'd be awesome if that `Binary files [a] [b] differ` line was nicer looking and better distinguished from the rest of the text

**Comment by scottchiefbaker** [comment]
Good catch... I think I was able to fix that, while adding some performance improvements as well. Try replacing your `lib/diff-so-fancy.pl` with this one: 

https://github.com/scottchiefbaker/diff-so-fancy/blob/binary-file-header/lib/diff-so-fancy.pl

If this does what you want it to I'll send a PR to get it landed in the next version.

**Comment by scottchiefbaker** [comment]
Sorry I got confused... there is another script that I updated also. Probably easiest to check out the whole repo and see if it fixes the problem or not.

https://github.com/scottchiefbaker/diff-so-fancy/tree/binary-file-header

**Comment by scottchiefbaker** [comment]
@alex have you had a chance to test this yet?

**Comment by alex** [comment]
Sorry, no I haven't.

**Comment by williamboman** [comment]
Looks good.

<img width="1550" alt="screenshot 2016-05-20 at 13 58 58" src="https://cloud.githubusercontent.com/assets/6705160/15427144/44720c94-1e93-11e6-82ab-4a2ae1f71aba.png">
<img width="1550" alt="screenshot 2016-05-20 at 14 00 17" src="https://cloud.githubusercontent.com/assets/6705160/15427145/44727422-1e93-11e6-98c0-1a31a9946076.png">

**Comment by scottchiefbaker** [comment]
Great. I'll get the PR ready.

**Comment by stevemao** [comment]
fixed by #156

**Comment by alex** [comment]
awesome, thanks.
